src/code/z_player_lib.c

```diff
@@ -837,21 +859,21 @@ Gfx* gPlayerSheath14DLs[2 * PLAYER_FORM_MAX] = {
     gLinkHumanSheathEmptyDL,
 };
 
-Gfx* gPlayerShields[][2] = {
-    { gLinkHumanHylianShieldWithMtxDL, gLinkHumanHylianShieldWithMtxDL },
-    { gLinkHumanMirrorShieldWithMtxDL, gLinkHumanMirrorShieldWithMtxDL },
+Gfx* gPlayerShields[] = {
+    gLinkHumanHylianShieldWithMtxDL,
+    gLinkHumanHylianShieldWithMtxDL,
+    gLinkHumanMirrorShieldWithMtxDL,
+    gLinkHumanMirrorShieldWithMtxDL,
```

Comment:
Do we know what the significance of this mtx is?

Reply:
It is used in `Player_PostLimbDrawGameplay`

```c

    } else if (limbIndex == PLAYER_LIMB_SHEATH) {
        if ((*dList1 != NULL) && (player->transformation == PLAYER_FORM_HUMAN) &&
            (player->currentShield != PLAYER_SHIELD_NONE) &&
            ((player->sheathType == PLAYER_MODELTYPE_SHEATH_14) ||
             (player->sheathType == PLAYER_MODELTYPE_SHEATH_15))) {
            OPEN_DISPS(play->state.gfxCtx);

            gSPDisplayList(POLY_OPA_DISP++, gPlayerShields[2 * ((player->currentShield - 1) ^ 0)]);

            CLOSE_DISPS(play->state.gfxCtx);
        }

        if (player->actor.scale.y >= 0.0f) {
            if ((player->rightHandType != PLAYER_MODELTYPE_RH_SHIELD) &&
                (player->rightHandType != PLAYER_MODELTYPE_RH_FF)) {
                static Vec3f sSheathLimbModelShieldOnBackPos = { 630.0f, 100.0f, -30.0f };
                static Vec3s sSheathLimbModelShieldOnBackZyxRot = { 0, 0, 0x7FFF };

                Matrix_TranslateRotateZYX(&sSheathLimbModelShieldOnBackPos, &sSheathLimbModelShieldOnBackZyxRot);
                Matrix_Get(&player->shieldMf);
            }
        }
```

Reply:
Sorry, I misunderstood.
Those dlists are named like that because they are just wrappers to the actual dlist:
```c

Gfx gLinkHumanHylianShieldWithMtxDL[] = {
    gsSPMatrix(&gLinkHumanHylianShieldMtx, G_MTX_NOPUSH | G_MTX_MUL | G_MTX_MODELVIEW),
    gsSPBranchList(gLinkHumanHylianShieldDL),
};
```

There are two other dlists that do something similar to this
```c

Gfx gLinkHumanRightHandHoldingHylianShieldDL[] = {
    gsSPDisplayList(gLinkHumanRightHandClosedDL),
    gsSPBranchList(gLinkHumanHylianShieldDL),
};

Gfx object_link_child_DL_01DC38[] = {
    gsSPDisplayList(object_link_child_DL_00FD08),
    gsSPBranchList(gLinkHumanHylianShieldDL),
};
```